runtime/stackstorm-1ppc/README.md

```diff
@@ -12,70 +12,54 @@ StackStorm Docker image that runs one st2 service per container.
 
 ## Getting started
 
-A sample `docker-compose.yml` file is located under `examples/stackstorm-1ppc` directory. Follow the instruction below to setup a running StackStorm instance which consists of containers that each are running individual st2 service.
+A sample `docker-compose.yml` file is located under `runtime/stackstorm-1ppc` directory.
+Follow the instruction below to setup a running StackStorm instance which consists of
+containers that each are running individual st2 service.
 
-1. Build `stackstorm/stackstorm-1ppc:latest` image
-
-Since 1ppc image is not officially released yet, you need to build the image by yourself.
-
-```
-(cd ../../images/stackstorm-1ppc \
-  && docker build -t stackstorm/stackstorm-1ppc:latest .)
-```
-
-Alternatively, you can use prebuilt image available at [shusugmt/stackstorm-1ppc](https://hub.docker.com/r/shusugmt/stackstorm-1ppc/). To do so simply replace `stackstorm:stackstorm-1ppc` with `shusugmt/stackstorm-1ppc` in `docker-compose.yml`.
-
-```
-sed -i -e 's/stackstorm\/stackstorm\-1ppc/shusugmt\/stackstorm\-1ppc/g' docker-compose.yml
-
-# for BSD sed (Mac)
-sed -i '' -e 's/stackstorm\/stackstorm\-1ppc/shusugmt\/stackstorm\-1ppc/g' docker-compose.yml
-```
-
-2. Generate .env files
+1. Generate .env files
 
 ```
 (cd ../../ && make env)
 cp -r ../../conf .
 ```
 
-3. Start containers
+2. Start containers
 
 ```
 docker-compose up -d
 ```
 
 Now you can access StackStorm Web UI.
 
-4. Register initial content
+3. Register initial content
```

Comment:
Isn't this handled by the entrypoint script?

Reply:
As coded now, it appears to be done automatically using k8s, but not using `docker-compose`.